catalog/transformer/catalog-transformer-xml/src/main/java/ddf/catalog/transformer/xml/adapter/AdaptedMetacard.java

```diff
@@ -160,7 +162,7 @@ protected void setMetacardType(MetacardType metacardType) {
     @XmlElement(name = "string", namespace = METACARD_URI, type = StringElement.class),
     @XmlElement(name = "stringxml", namespace = METACARD_URI, type = StringxmlElement.class)
   })
-  protected List<Attribute> getAttributes() {
+  protected Set<Attribute> getAttributes() {
     return attributes;
```

Comment:
❓ Should this be a defensive copy?

Reply:
Can't be. Serialization depends upon having access to the underlying structure. Making a defensive copy breaks a lot of unit tests and work require a deal of rework if it was even possible in the first place.